src/Microsoft.Health.Fhir.Liquid.Converter/Hl7v2/Models/UnusedHl7v2Component.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Hl7v2.Models
+{
+    public class UnusedHl7v2Component
+    {
+        public UnusedHl7v2Component(int index, string value)
+        {
+            Index = index;
```

Comment:
The end position is the index plus the length of value, right?

Reply:
The source code is updated. The start position is `Offset` and the end position is `Offset + Value.Length`.

Reply:
I feel you can provider readonly properties for start and end positions 

---
In reply to: [540747689](https://github.com/microsoft/FHIR-Converter/pull/123#discussion_r540747689) [](ancestors = 540747689)

Reply:
Thanks, added end index in the output.